ANONYMITY_NETWORKS.md

```diff
@@ -222,3 +222,71 @@ leak information to hidden services if done improperly.
 At the current time, if users need to frequently make transactions, I2P/Tor
 will improve privacy from ISPs and other common adversaries, but still have
 some metadata leakages to unknown hidden service operators.
+
+# Example: Configure a remote, linux daemon that you can connect to via tor from a linux or mac laptop
```

Comment:
This whole section/example is about connecting to a remote daemon over Tor. The rest of the document is specifically about how your _daemon_ can broadcast transactions to other _daemons_ on the network over tor/i2p. They have different (although related) use cases. I'm not against adding this instruction in, just concerned it is not calling this dissection out enough. This whole section doesn't even include what the rest of document is outlining.

Reply:
My suggestion would be twofold. 1) clearly structure the document to cover both use cases. I.e. describe both cases and how they relate. 2) incorporate both use cases into one example section. 

As it is in this PR, it's confusing. The document starts (as was it's original purpose), discussing the features for daemon to daemon (p2p) communication over tor/i2p, but then gives a big example of *wallet* to daemon over tor.

Reply:
Yes, I see what you mean.
This sounds like a pretty significant restructuring (which I'm happy to do).
But, considering it is already quite long, would it make sense to separate examples into another "Cookbook" or "Examples" document?  I could add an example using the GUI with `torsocks` (just discovered that works) and examples which are more specific to running an open node as an eepsite / hidden service, etc.

Reply:
My points still stand with your latest edits here. This document has become Frankenstein'ed.

Also worth noting, the example you are trying to mix in with this document is kind of already covered in https://github.com/monero-project/monero#using-tor. Why not just update/elaborate that instead?

Reply:
Okay.  Thanks for the feedback.